tgui/packages/common/vector.ts

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Can you add an explicit return type to these functions please? For compiler efficiency